### DISCUSSION TITLE
Make issues status in private project available to public

### DISCUSSION THREAD

**Discussion by cosmic-flood** [community/community#38048] [Projects and Issues]
Our project is private, public issue's project information is not shown to public (only available to private team). We want the public users be also able to see it. 

<img src="https://user-images.githubusercontent.com/68597908/199407127-22b39f5b-f54a-4cfc-9ef8-b8b81480b9b6.png" width="500">

Is it possible？Do we have alternative ways?

**Comment by david-engelmann** [comment]
You could create a public repository just for submitting issues. Docker currently has a similar setup for submitting bug for the various platforms. Check it out!

https://github.com/docker/for-win
https://github.com/docker/for-mac
https://github.com/docker/for-linux
